src/stateMachines/common.test.ts

```diff
@@ -110,4 +114,9 @@ describe('Common', () => {
     expect(enterpriseAssistantListener.promises[clientId].resolve).toBeDefined()
     expect(enterpriseAssistantListener.promises[clientId].reject).toBeDefined()
   })
+  it('should timeout on no reponse', async () => {
+    const wsmanPromise = invokeWsmanCall(context, 1)
+    await devices[clientId].reject(GATEWAY_TIMEOUT_ERROR)
```

Comment:
rejecting the promise does not allow the timeout to occur. Assume the test was taking too long? here's a snippet changing the timeout value instead and letting the timeout reject the promise as intended
```javascript
    Environment.Config.delayTimer = 1
    const wsmanPromise = invokeWsmanCall(context, 1)
    await expect(wsmanPromise).rejects.toEqual(GATEWAY_TIMEOUT_ERROR)
`